docs/source/import_export/main.rst

```diff
@@ -142,16 +141,43 @@ Let's have a look at the contents of the metadata.json::
             },
             "Group": {
                 "description": {},
-                "name": {},
+                "label": {},
                 "user": {
                     "related_name": "dbgroups",
                     "requires": "User"
                 },
                 "time": {
                     "convert_type": "date"
                 },
-                "type": {},
+                "type_string": {},
                 "uuid": {}
+            },
+            "Log": {
```

Comment:
Since you will have to do some minor updates anyway, might as well remove this here and add it with the correct PR on logs.